controllers/powerprofile_controller_test.go

```diff
@@ -670,10 +686,7 @@ func TestPowerProfile_Reconcile_SharedProfileDoesNotExistInLibrary(t *testing.T)
 		}
 
 		_, err = r.Reconcile(context.TODO(), req)
-		if err != nil {
-			t.Error(err)
-			t.Fatalf("%s - error reconciling object", tc.testCase)
-		}
+		assert.ErrorContains(t, err, "maximum or minimum frequency value cannot be below 1000")
```

Comment:
```suggestion
		assert.ErrorContains(t, err, "max or min frequency value cannot be below 1000")
```

Reply:
Will address all changes to controllers/controller tests in the upcoming release